### PR TITLE
[CON-212] feat: kit (Write)

### PR DESCRIPTION
**Pull request by sanjay-a01**
### Write Functionality Response

image

image

image

### Test Result for Write Functionality in Kit Connector

image

### CODE REVIEW

File: providers/kit/write.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			Success: true,
+		}, nil
+	}
+
+	// Write response has a reference to the resource but no payload data.
+	return constructWriteResult(config.ObjectName, body)
+}
+
+func constructWriteResult(objName string, body *ajson.Node) (*common.WriteResult, error) {
+	obj := naming.NewSingularString(objName)
```
Comment (Cobalt0s) [inline providers/kit/write.go:57]: Maybe call the string method in here and more decriptive name.
naming.NewSingularString(objName).String()

obj => objectPath, jsonPath, nodePath, objectFieldName, objectNodeLocation
I honestly don't know how to name it.

**Review by Cobalt0s — APPROVED** [review]
LGTM

The only issue as in other PRs, remove read/metadata methods to make this PR dedicated. My review was focused on write.

File: providers/kit/objectNames.go [hunk 2]
```diff
@@ -16,7 +16,23 @@ const (
 )
 
 var supportedObjectsByRead = datautils.NewSet( //nolint:gochecknoglobals
+	objectNameBroadCasts,
```
Comment (Cobalt0s) [inline providers/kit/objectNames.go:19]: These should be added as part of the `Read PR`.

**Review by Cobalt0s — APPROVED** [review]
The write functionality LGTM.

**Review by eberle1080 — APPROVED** [review]

**Pull request merged**